libethereum/BlockChainSync.cpp

```diff
@@ -261,10 +261,15 @@ void BlockChainSync::syncPeer(std::shared_ptr<EthereumPeer> _peer, bool _force)
 
     if (_force || _peer->m_totalDifficulty > syncingDifficulty)
     {
+        LOG(m_logger) << "Discovered new highest difficulty";
```

Comment:
It's not clear that the log verbosity is.

Reply:
`m_logger` definition is
```
    Logger m_logger{createLogger(VerbosityDebug, "sync")};
```
So it's DEBUG

Reply:
Run with e.g. `-v 3 --log-exclude-channels net discov` to be able to see it

Reply:
I have a small problem with this. I'd like this message to be INFO. It's not good idea to bind channel with verbosity.

Reply:
We can make it INFO, I'm just afraid that it could lead to many such messages (every time `syncPeer` is called (that's not only when new peer connects) and highest difficulty changes, which is often in live network). Also `new highest difficulty` sounds quite low-level for default log level which I think should be more average-user-oriented.

Maybe output to INFO only `Starting full sync` below and leave this in DEBUG?

Reply:
Yes, INFO "Starting full sync" sounds good.